spec/SPEC-annex-C-source-embedding.md

```diff
@@ -0,0 +1,95 @@
+## Annex C
+
+Annex C contains a method of embedding an Input Manifest Identifier into source code files files.
+
+### Embedded Input Manifest Identifier
+
+Most source code files are hand coded by humans.  Some however are generated from other input(s) by a build tool.
+
+A build tool outputing a source code file may embed the Input Manifest Identifier for the output source code file into
+the output source code file by adding a comment line containing a string of the form:
+
+``` OmniBOR-Input-Manifest-ID: [ ${comma separated list of Input Manifest Identifier URIs} ]
+```
+
+For a file with C commenting semantics (like C, C++, Java, Go, etc) a concrete example might be:
+
+```// OmniBOR-Input-Manifest-ID: [ gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64, gitoid:blob:sha256:09c825ac02df9150e4f93d12ba1da5d1ff5846c3e62503c814aa3a300c535772 ]
+```
```

Comment:
How would this interact with 80 char line limits in some linters?  

Should the opening and closing square brackets be present even if there is only a single hash?

Reply:
Consensus is that we should always have the opening and closing square brackets even if there is only a single hash.